Game/Src/TestEnemyMovement.gd

```diff
@@ -120,8 +121,13 @@ func _process(delta):
 				emit_signal("hero_starts_fighting")
 
 	elif current_game_status == game_status.FIGHTING:
-		GlobalVariables.glory += closest_enemy.stats.max_health
-		GlobalVariables.damage += closest_enemy.stats.glory_win
+		emit_signal("glory_increased",closest_enemy.stats.glory_win)
+		var playerHealth = characterStats.get_health()
+		print('current health ',playerHealth)
+		print('attack ',closest_enemy.stats.attack)
+		characterStats.set_health(playerHealth-closest_enemy.stats.attack)
+#		GlobalVariables.glory += closest_enemy.stats.glory_win
```

Comment:
Can we delete these? They serve no purpose it seems.
Also, the debug prints are still here

Reply:
Only the prints, set_health is useful